Index: declare.h
===================================================================
--- declare.h.orig	2004-01-30 00:13:50.000000000 +0200
+++ declare.h	2007-07-17 08:44:05.000000000 +0300
@@ -15,12 +15,14 @@
 
 #else /* AMIGA */
 
-typedef unsigned short USHORT;
-typedef short          SHORT ;
-typedef unsigned short UWORD ;
-typedef short          WORD  ;
-typedef unsigned long  ULONG ;
-typedef long           LONG  ;
+#include <stdint.h>
+
+typedef uint16_t       USHORT;
+typedef int16_t        SHORT ;
+typedef uint16_t       UWORD ;
+typedef int16_t        WORD  ;
+typedef uint32_t       ULONG ;
+typedef int32_t        LONG  ;
 
 #endif  /* !AMIGA */
 
